client/cmd/main.go

```diff
@@ -41,13 +41,34 @@ func main() {
 		os.Exit(1)
 	}
 
-	// step 3: analyze result and return to CI the final result.
+	// step 3: analyze result
 	analysis.AnalyzeResult(huskyAnalysis)
 
+	// step 4: print output
+	formatOutput := ""
+	if len(os.Args) > 1 {
+		formatOutput = "JSON"
+	}
+	err = analysis.PrintResults(formatOutput)
+	if err != nil {
+		fmt.Println("[HUSKYCI][ERROR] Printing output:", err)
+		os.Exit(1)
+	}
+
+	// step 5: block developer CI if vulnerabilities were found
 	if types.FoundVuln == true {
 		os.Exit(1)
 	}
 
-	os.Exit(0)
+	if types.FoundInfo == true {
+		fmt.Printf("[HUSKYCI][*] Some low/info issues were found :|\n")
+		fmt.Println()
+		os.Exit(0)
+	}
 
+	if len(os.Args) < 1 {
```

Comment:
Why did you choose to check `len(os.Args)` here?  I think if code reaches this stage means that no vulnerabilities were been found, regardless of the `os.Args`. Or did I miss something? 🙃

Reply:
Sounds Ok, @Krlier. Thanks for your explanation! 😎